src/dummyClient/dummyClient.js

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+
+var socket = require('socket.io-client')('http://localhost:3000');
+socket.on('connect', function(){
```

Comment:
Suggest using arrow function instead of function declaration.  Reasons below. 

```
socket.on('connect', function() {
   // Your code
});
```

Turns into:

```
socket.on('connect', () => {
    // Your code
});
```

Reasons: 
Function declarations (those that use the word function), when called create a new context, so if you want to access the `this` variable from the calling function you have to assign it to a new variable in the calling function and use that instead.   Arrow functions don't create a new context so you don't have to go through that rigamarole.